assets/src/js/admin/reports/components/not-found-notice/index.js

```diff
@@ -0,0 +1,64 @@
+// Dependencies
+import { Fragment } from 'react';
+const { __ } = wp.i18n;
+
+// Store-related dependencies
+import { useStoreValue } from '../../store';
+
+// Styles
+import './style.scss';
+
+const NotFoundNotice = ( { version } ) => {
+	// Get 'period' object from the store
+	const [ {}, dispatch ] = useStoreValue();
+
+	// Dispatch SET_RANGE action
+	const setRange = ( range ) => {
+		dispatch( {
+			type: 'SET_RANGE',
+			payload: {
+				range,
+			},
+		} );
+	};
```

Comment:
Store actions in the reducer.

Reply:
This component has actually been removed, as a result of rethinking the NotFound popups. Instead of a NotFound notice being rendered for periods when there is donaiton data on the install, but not data for the given period, we show nothing. Ref: https://github.com/impress-org/givewp/commit/0319b640e850e8908c5e8928a3f886410cb2156b